util/interactive_helpers.tcl

```diff
@@ -0,0 +1,74 @@
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+# Some helpful TCL procedures for use in an interactive Vivado session.
+# Activate when launching Vivado using:
+#   vivado -source util/interactive_helpers.tcl ...
+# or later in the Vivado TCL shell using:
+#   source util/interactive_helpers.tcl
+
+# Change Selection
+proc cs args {select_objects {*}$args}
+
+# TIMing report - setup
```

Comment:
Nit: Timing

Reply:
The capitalisation? I did it like that to highlight the link between the short-hand proc name and the longer-form description to aid memorisation.

Reply:
Ok, that's fine then.